api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
```

Comment:
It is hidden because a developer initially approaching the project needs only run `./project.rb` to get a sense of what he/she can do, and only needs to start peeking in to the `.project` folder when he/she needs to make changes to that (likely much later in the ramp-up process than changing code, for example). It also makes tab-completion of `./project.rb` slightly more convenient. I am not in love with this naming convention, so please do suggest alternatives.